src/LoginButton/index.tsx

```diff
@@ -10,6 +10,9 @@ const AffinidiLoginButton = (props: AffinidiLoginTypeProps) => {
   async function logInHandler() {
     const data = await getAuthUrl();
     window.location.href = data.authorizationUrl;
+    if (props.onSuccess) {
+      props.onSuccess()
+    }
   }
```

Comment:
Hi @kidscoots101 / Caleb,

Thank you for your interest and initiative. I appreciate your suggestion to add a success callback, but unfortunately, in this scenario, the code after `window.location.href` will not be executed because the current document will be no longer active as the new page is loading.

However, if needed, we can invoke a success callback just before redirecting

Please let me know if you have any other questions or suggestions.

Reply:
Hi @kamarthiparamesh / Paramesh, 

Thanks for your reply. Will it be better to put the `window.location.href` part with the `if (props.onSuccess) { props.onSuccess() }` code?

This case it would only redirect if the login is successful.

I've committed it already so you can take a look! :)

Reply:
Hi @kamarthiparamesh, any updates? :)